ansible/group_vars/all

```diff
@@ -589,14 +611,34 @@ mnohub:
 nex:
   skip: true
   redis:
+    skip: false
     name: "{{ environment_name }}-nex-redis"
     node_type: cache.t2.micro
     num_nodes: 1
     endpoint: "{{ environment_name }}-nex-redis.{{ dns.base }}"
     cache_port: 6379
     cache_engine_version: 3.2.4
     cache_parameter_group: "default.redis3.2"
+  rds:
+    skip: false
```

Comment:
we may want to set this parameter (and all other related params) to `{{ rds.skip }}` instead of `false` so as to keep things backward compatible